### PR TITLE
New Homepage Commissioners section styles

### PR DESCRIPTION
**Pull request by xtine**
Support for https://github.com/18F/fec-cms/pull/662

### CODE REVIEW

File: scss/layout/_grid.scss [hunk 1]
```diff
@@ -119,14 +130,32 @@
...
   }
 
-  .slab .grid {
-    .grid__item:nth-last-child(-n+2) {
-      border-bottom: none;
-      margin-bottom: 0;
+  .slab {
+    .grid {
+      h3 {
+        line-height: 1.5;
```
Comment (noahmanger) [inline scss/layout/_grid.scss]: Feels a little weird to have this property nested in `.slab > .grid`. Should it be a more generalized class or applied to all h3s?

File: scss/layout/_grid.scss [hunk 2]
```diff
@@ -119,14 +130,32 @@
...
+      }
+    }
+
+    .grid--2-wide {
+      .grid__item:nth-last-child(-n+2) {
```
Comment (noahmanger) [inline scss/layout/_grid.scss]: I didn't know about the `-n` trick.

File: scss/layout/_grid.scss [hunk 3]
```diff
@@ -119,14 +130,32 @@
     }
   }
 
-  .slab .grid {
-    .grid__item:nth-last-child(-n+2) {
-      border-bottom: none;
-      margin-bottom: 0;
+  .slab {
```
Comment (noahmanger) [inline scss/layout/_grid.scss]: Can this be combined with the `.slab > .grid` nest above?

File: scss/layout/_grid.scss (hunk 1 above)
Comment (xtine) [inline scss/layout/_grid.scss]: Yeah I'm not quite sure where to put this because I don't want to affect the other existing `h3`.

**Comment by ccostino** [comment]
Since this is approved and remarked, I'm merging in - we can continue to improve in the release branch if need be!

**Pull request merged**